query/src/main/scala/filodb/query/exec/ScalarFixedDoubleExec.scala

```diff
@@ -50,17 +52,28 @@ case class ScalarFixedDoubleExec(id: String,
                        queryConfig: QueryConfig)
                       (implicit sched: Scheduler,
                        timeout: FiniteDuration): Task[QueryResponse] = {
+    val execPlan2Span = Kamon.spanBuilder(s"execute-${getClass.getSimpleName}")
+      .asChildOf(Kamon.currentSpan())
```

Comment:
Just search for `Kamon.spanBuilder` in code base and add `.asChildOf(Kamon.currentSpan())` where applicable. I hate that Kamon wouldnt pick this up without us needing to tell it.